storage/google/cloud/storage/client.py

```diff
@@ -475,7 +475,7 @@ def list_blobs(
             versions=versions,
             projection=projection,
             fields=fields,
-            client=self
+            client=self,
```

Comment:
Please rebase against the head of `master`, which already has this change (and the others below) from PR #8831.